client/components/line-chart/index.js

```diff
@@ -419,7 +444,9 @@ class LineChart extends Component {
 					`circle.line-chart__line-point-color-${ dataSeriesIndex }`
 				);
 				selectedPoints.attr( 'r', Math.floor( POINTS_SIZE * POINT_HIGHLIGHT_SIZE_FACTOR ) );
-				this.setState( { selectedPoints: selectedPoints.nodes() } );
+				this.setState( {
+					selectedPoints: selectedPoints.nodes(),
+				} );
```

Comment:
Why the formatting change here?

Reply:
Good catch, looks like my code editor caused these formatting changes.